source/codegen/templates/service.h.mako

```diff
@@ -66,9 +67,10 @@ private:
 % for enum in enums_to_map:
 <%
   enum_value = service_helpers.python_to_c(enums[enum])
+  map_prefix = enum[:-len("Mapped")] if enum.endswith("Mapped") else enum
```

Comment:
This condition is everywhere!!

1. Do we really have to check ends with `Mapped`? (can we just use the name as-is?)
2. Do we really have to strip `Mapped` off the name?
3. Can we put code with logic based on `Mapped` in a python helpers file so that it's shared and close together?

Reply:
I think the problem is that we're adding that suffix to anything with `generate-mappings`. The code generator shouldn't be doing that.

Reply:
> When you say "add the suffix to all enum names" you are referring to the automatically-generated merged attribute enums, right? 

I was actually talking about all enums tagged with `generate-mappings`. But based on our discussion on other thread about whether or not we should add `Mapped` suffix automatically, I can do it just for automatically-generated merged attribute values enums only.